docs/development.md

```diff
@@ -39,23 +39,33 @@ In addition, the command will also *minify* the JavaScript files to reduce the s
 
 #### Starting the dev server
 
-Run the following command:
+Run the following command and wait until the task exits with a `BUILD SUCCESSFUL`:
```

Comment:
``To start the server in the background, run the following command and wait until the task exits with a `BUILD SUCCESSFUL`:``